editor/components/block-mover/index.js

```diff
@@ -24,13 +24,13 @@ import { selectBlock } from '../../store/actions';
  * Module constants
  */
 const upArrow = (
-	<svg tabIndex="-1" width="18" height="18" xmlns="http://www.w3.org/2000/svg" aria-hidden role="img" focusable="false">
+	<svg className="dashicon" width="18" height="18" xmlns="http://www.w3.org/2000/svg" aria-hidden role="img" focusable="false">
```

Comment:
Asked about the intent of these icons at https://github.com/WordPress/gutenberg/pull/5263#issuecomment-370635173, but I generally would discourage pretending to inherit styles from a separate component.

Reply:
I'd agree best would be to have them integrated into Dashicon